src/store/wallet/screen/screenReducers.js

```diff
@@ -21,9 +21,7 @@ function onOpen(state, action) {
 
 function onError(state, action) {
   if (action.type === 'SCREEN/ERROR') {
-    return state
-      .set('error', action.message)
-      .set('dialog', null).set('dialogItem', null);
```

Comment:
Not sure why it was setting this to null. Maybe this was another bug fix? Should I put this back?